dali/base/dadfs.cpp

```diff
@@ -7929,6 +7930,130 @@ IDistributedSuperFile *CDistributedFileDirectory::createSuperFile(const char *_l
 }
 
 // MORE: This should be implemented in DFSAccess later on
+IDistributedSuperFile *CDistributedFileDirectory::createSuperFile(IPropertyTree *tree, IUserDescriptor *user, bool interleaved, bool ifdoesnotexist, IDistributedFileTransaction *transaction)
+{
+    StringBuffer superName;
+    Owned<IDistributedSuperFile> superfile;
+
+    if (!tree->hasProp("@name"))
+        throw MakeStringException(-1, "createSuperFile: There is no SuperFile name in XML");
```

Comment:
trivial: this test is unnecessary really, the test below would catch it and is sufficient.